backend/build.gradle.kts

```diff
@@ -23,7 +23,7 @@ extra["snippetsDir"] = file("build/generated-snippets")
 
 dependencies {
 	implementation("org.springframework.boot:spring-boot-starter-mail")
-	implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation("org.springframework.boot:spring-boot-starter-web")
```

Comment:
Now spaces and tabs are inconsistent in this file. Since this file uses tabs for the rest, this line should also use tabs. 
How we deal with that in general is another discussion, especially since frontend and backend currently use different styles, we should unify this, see #18